packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -55,6 +55,14 @@ const
 	JS = 'JS',
 	Native = 'Native';
 
+/**
+ * The base version of the Moonstone virtual list component.
+ *
+ * @class VirtualListBase
```

Comment:
We already defined `VirtualListBase` in this file.

```
/**
 * A Moonstone-styled base component for [VirtualList]{@link moonstone/VirtualList.VirtualList} and
 * [VirtualGridList]{@link moonstone/VirtualList.VirtualGridList}.
 *
 * @class VirtualListBase
 * @memberof moonstone/VirtualList
 * @extends ui/VirtualList.VirtualListBase
 * @ui
 * @private
 */
const VirtualListBase = VirtualListBaseFactory(JS);
VirtualListBase.displayName = 'VirtualListBase';
```

Reply:
I missed that.  It's private, I guess that's why the docs didn't show up.  We can revert that, I guess.  I'll take another stab at it later.

Reply:
I made `VirtualListCore`, `VirtualListBase` public. Then the `role`, `itemRenderer`, and `isItemDisabled` of `VirtualList` became visible in Enact API documentation.

Reply:
`VirtualListBase` and `VirtualListBaseNative` are based on `VirtualListCore`.
`VirtualList` and `VirtualGridList` are based on `VirtualListBase`,
`VirtualListNative` and `VirtualGridListNative` are based on `VirtualListBaseNative`.

So all common virtual list related props should be defined in `VirtualListCore`.